src/vma/dev/ring_simple.cpp

```diff
@@ -869,6 +869,7 @@ bool ring_simple::rx_process_buffer(mem_buf_desc_t* p_rx_wc_buf_desc, transport_
 	case IPPROTO_IGMP:
 	{
 		struct igmp* p_igmp_h= (struct igmp*)((uint8_t*)p_ip_h + ip_hdr_len);
+		p_igmp_h = p_igmp_h; /* to supress warning in case VMA_OPTIMIZE_LOG */
```

Comment:
In order to keep consistency with other parts of the code please consider using:
```code
NOT_IN_USE(p_igmp_h); /* to supress warning in case VMA_OPTIMIZE_LOG */
``` 
instead of
```code2
p_igmp_h = p_igmp_h; /* to supress warning in case VMA_OPTIMIZE_LOG */
```